Typescript error

Since `authenticate` is not defined in the index.d.ts file, `tsc` and `tslint` is throwing error 
```
error TS2345: Argument of type 'import("/node_modules/passport-custom/lib/index").Strategy' is not assignable to parameter of type 'import("/node_modules/@types/passport/index").Strategy'.
  Property 'authenticate' is missing in type 'import("/node_modules/passport-custom/lib/index").Strategy' but required in type 'import("/node_modules/@types/passport/index").Strategy'.
21 use(new Strategy((req: Request, done: VerifyCallback) => {
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
22     done(null, {});
   ~~~~~~~~~~~~~~~~~~~
23 }));
   ~~

  ../node_modules/@types/passport/index.d.ts:91:9
    91         authenticate(this: StrategyCreated<this>, req: express.Request, options?: any): any;
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'authenticate' is declared here.
```

Possible Solution:

add this line in `index.d.ts`

```js
export declare class Strategy extends PassportStrategy {
    constructor(verify: VerifyCallback);
    authenticate(this: StrategyCreated<this>, req: express.Request, options?: any): any;
}
```

--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -11,6 +11,7 @@ export interface VerifiedCallback {
 
 export declare class Strategy extends PassportStrategy {
     constructor(verify: VerifyCallback);
+    authenticate(req: Request, options?: any): any;
 }
 
 

